content/nextflow-on-aws-batch/40_nextflow202/10_setup.md

```diff
@@ -8,35 +8,36 @@ weight: 10
 
 Now that we have created queues and compute environments, we can wire them into Nextflow.
 
-{{% notice info %}}
-Please note that we are creating a config that does not hold a container name nor a AWS_REGION. We are going to change those values using `sed`.
-{{% /notice %}}
+Nextflow will evaluate a `nextflow.config` file next to the script we are executing (which would be the file in the current directory) and also fall back to `$HOME/.nextflow/config` for additional configuration. As we are going to use the latter one when using AWS Batch squared we are changing both.
```

Comment:
> when using AWS Batch **squared** we are changing both

Is squared a typo ? I don't get it.

Reply:
lol, no - it is not. I admit it does not really contribute to make it less confusing. 
NF is supervising the execution of the workflow - no matter which executor (Batch, docker, local, K8s) is used. NF+batch-executor is the local run.
NF submitted as a batch-job itself, which submits more jobs to another queue is called batch-squared...

Reply:
> As we are going to use the latter one when using AWS Batch queue `batch-squared` we are changing both.

Does that sound better to you ?